### DISCUSSION TITLE
Change haute42 S16 firmware

### DISCUSSION THREAD

**Discussion by aduhglupa** [OpenStickCommunity/GP2040-CE#980] [Q&A]
Hi, I just bought a haute42 S16. and when I check on Web Configurator, It comes with Pico (GP2040-CE_0.7.5_Pico.uf2) v0.7.5.

Can I update the firmware directly to GP2040-CE_0.7.8_Haute42COSMOX.uf2? Or should I update to Pico version 0.7.8 instead?

**Comment by mikepparks** [comment]
Some Haute42 models prior to 0.7.6 used the Pico build, but would apply settings on top of it.  In this instance, I would suggest using the [Flash Nuke](https://gp2040-ce.info/installation#flash-nuke-process) process and install the updated firmware in its place.

**Comment by TheTrainGoes** [comment]
@aduhglupa if you need additional help with your Haute42 device please ask on their Discord directly:
https://discord.com/invite/hcCyQRffbU
